sharktank/sharktank/utils/hf_datasets.py

```diff
@@ -247,6 +247,40 @@ def alias_dataset(from_name: str, to_name: str):
     ),
 ).alias_to("mixtral_8x7b_q8_0_gguf")
 
+Dataset(
```

Comment:
While you are at it, might as well add all llama2 and llama3 variants available on amd-shark HF repo, if we support them on sharktank.

Reply:
I will add them to the amd-shark HF repo later.